dandi/dandiapi.py

```diff
@@ -123,7 +122,9 @@ def send_request(
         url = self.get_url(path)
 
         # Make the request, passing parameters and authentication info
-        _headers = headers or {}
+        _headers = dict(self._headers)
+        if headers:
+            _headers.update(_headers)
```

Comment:
The second `_headers` should not have a leading underscore.

Reply:
d'oh, right -- thanks! will fix